common/lib/xmodule/xmodule/html_module.py

```diff
@@ -446,10 +446,8 @@ def get_html(self):
                 return self.data.replace("%%USER_ID%%", self.system.anonymous_student_id)
             return self.data
         else:
-            course_updates = self.ordered_updates()
```

Comment:
It would be better to not touch this file if possible, to ensure that the behavior without the Waffle flag stays the same. I'd prefer to see the changes isolated into the course_experience Django app.